lib/plugins/aws/provider/awsProvider.test.js

```diff
@@ -123,12 +124,10 @@ describe('AwsProvider', () => {
       awsProvider.sdk = {
         S3: FakeS3,
       };
-      awsProvider.request('S3', 'error', {}, 'dev', 'us-east-1')
+      awsProvider.request('S3', 'error', {})
         .then(data => {
-          // eslint-disable-next-line no-unused-expressions
-          expect(data).to.exist;
-          // eslint-disable-next-line no-unused-expressions
-          expect(first).to.be.false;
+          expect(data).to.exist; // eslint-disable-line no-unused-expressions
```

Comment:
I tried but couldn't ... the test case is that simple that it just needs an expect